src/components/vr-scatter-plot/vr-scatter-plot.component.ts

```diff
@@ -44,13 +48,12 @@ export class VRScatterPlotComponent<T extends Meta>implements OnInit, OnChanges,
     .subscribe(dataset => {
       // componentMetas is initialized to different dataset metas - will help funnel dataset
       this.componentMetas = dataset.metas;
-      // dataset.metas[0].type = 'tabbed' and dataset.metas[1] = 'line'
-      this.meta = dataset.metas[0];
-      if (this.isTabbed()) {
-        this.meta2 = (this.meta as any).metas[0];
-        console.log(this.meta2);
-      } else {
-        this.meta2 = this.meta[0];
+      // dataset.metas[0].type = 'tabbed' and dataset.metas[1] = 'line' if chosen UserWhiteNoise
+      // dataset.metas[0] - 'line' if Dummy chosen
+      if (dataset.metas[0].type === 'tabbed') {
+        this.datasetPref = (dataset.metas[0] as any).metas[0];
```

Comment:
Could you use a more specific type than `any`.

Reply:
I went ahead an imported `TabbedChartsMeta` from dataset metas and then casted as so